paasta_tools/deploy_chronos_jobs

```diff
@@ -1,4 +1,4 @@
 #!/bin/bash
 if am_i_mesos_leader >/dev/null; then
-  list_chronos_jobs | shuf | xargs -n 1 -r -P 5 setup_chronos_job
+  list_chronos_jobs | tr ' ' '\n' | shuf | xargs -n 1 -r -P 5 setup_chronos_job
```

Comment:
This is... fine I think. What if list_chronos_jobs (and list_marathon_jobs) did one per line anyway?